tests/basic_hybrid.py

```diff
@@ -0,0 +1,67 @@
+"""
```

Comment:
Perhaps rename to "test_basic_hybrid" since other test files have "test_bla" naming?